en/docs/trust/wso2-public-cloud-data-protection-faq.md

```diff
@@ -0,0 +1,441 @@
+
+Additionally, application developers have the flexibility to implement their own encryption mechanisms to safeguard their business data. This allows developers to tailor security solutions to meet specific organizational or regulatory requirements, ensuring maximum control over data protection strategies.
+
+### How is data encrypted in transit?
+
+WSO2 encrypts data in transit by leveraging TLS 1.2. Furthermore, all CDP and PDP communications within the cluster are encrypted using Cilium WireGuard encryption. WireGuard uses the ChaCha20 encryption algorithm with a 256-bit key.
+
+### How is data encrypted at rest?
+
+WSO2 ensures data at rest is encrypted using cloud service provider (CSP) technologies, including disk and database encryption. These mechanisms typically utilize AES-256 encryption to provide robust security.
+
+### How are Secrets encrypted?
+
+Secrets are encrypted using cloud service provider-managed keys and securely stored in key management services, such as Key Vaults. These services ensure robust encryption, centralized management, and controlled access to sensitive data.
+
+### Are application service users stored securely?
+
+Choreo ensures that application service users are stored and managed securely by leveraging trusted and industry-standard identity providers.
+
+* **User Authentication**: Choreo primarily authenticates users through well-established social logins, such as Google, GitHub, and Microsoft, ensuring the security of user credentials. Additionally, Choreo supports [Enterprise ID logins](https://wso2.com/choreo/docs/administer/configure-enterprise-login/), allowing seamless integration with your organization's identity provider for Single Sign-On (SSO). This means the complexity of password management is handled by the chosen identity provider, not Choreo itself, minimizing security risks.
+
+* **API Consumer Authentication and Authorization**: Choreo uses [Asgardeo as the default external identity provider](https://wso2.com/choreo/docs/authentication-and-authorization/secure-api-access-with-asgardeo/) (IDP), which can be used to securely manage end-user identities. If you prefer to use an external IDP like [Azure Active Directory](https://wso2.com/choreo/docs/authentication-and-authorization/secure-api-access-with-azure-ad/), Choreo allows flexible integration, ensuring that user management and authentication are securely handled by the respective external IDP.
+
+* **User Tokens**: The Choreo Console stores the tokens in the browser's session storage and uses them to authenticate the user with the backend services. Choreo CLI persists the tokens in an encrypted file in the local machine's file system. The key to decrypt the tokens is stored in the system's Keychain. The Choreo VSCode Extension uses the Choreo CLI to manage the tokens and invoke backend services.
```

Comment:
Do we need to talk about how the user tokens are stored under `Are application service users stored securely?` ? @manjulaRathnayaka